packages/story-editor/src/components/colorPicker/basicColorPicker.js

```diff
@@ -97,7 +97,7 @@ function BasicColorPicker({
           allowsGradient={allowsGradient}
           aria-labelledby="colorpicker-default-colors-title"
         />
-        {allowsSavedColors && savedColors.length && (
+        {allowsSavedColors && savedColors.length > 0 && (
```

Comment:
When I was resetting the site I saw this:
![Screenshot 2021-09-27 at 16 30 45](https://user-images.githubusercontent.com/21036224/134979716-480c4065-04f1-42ab-8fe1-95f16e85588f.png)